config/proto/targets.proto

```diff
@@ -34,15 +47,17 @@ message TargetsDef {
 
 // REQUIRED, when using Ceph as the target system.
 // Comparable functionality is handled in the Go client library for GCS.
-message CephConnectionDetails {
-  int32 port = 1;  // Port for connecting to Ceph S3 API
-  
+message ConnectionDetails {
+  // REQUIRED for Ceph S3 API
+  // Port for connecting to Ceph S3 API
+  int32 port = 1;
+
   // REQUIRED for Ceph S3 API
   // GCS uses service account credentials instead.
   string api_key = 2;
 }
 
 // Targets allows the user to define multiple targets.
 message Targets {
-  repeated TargetDef targets = 1;
+  repeated TargetDefinition targets = 1;
```

Comment:
You also need to change the definition of this message on line 28 (if I'm reading the github review properly)

Reply:
Yup, I did that in a more recent commit as I needed to change it across multiple files.